.eslintrc.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @type {import("eslint").Linter.Config}
+ */
```

Comment:
What is this? @Keyrxng @gentlementlegen

Reply:
I didn't know they were explicitly forbidden as `jest.config.js` is a common config file across UBQ repos but I can refactor to one of the other accepted extensions

Reply:
Jest is the exception if I recall correctly due to limitations 

You changed this file format in this pull

Reply:
https://github.com/ubiquity/ts-template/pull/23#discussion_r1598226467

Since it's a configuration file, didn't see much of the problem as long as it is typed. Can revert to `json`.

Reply:
@0x4007 https://github.com/ubiquity/ts-template/pull/44